src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -123,6 +123,7 @@ class Dummy3(Dummy2):
 # ------------- Tests for built-ins methods and arguments ------------------- #
 
 
+# pylint: disable=invalid-name
```

Comment:
This should not be required as long as you run pytest from within the `src/psyclone/tests` directory. Please could you double-check?

Reply:
Apologies, I added this to filter out all of the `invalid-name` warnings from `pylint` to see if there were any warnings I had created. It is definitely not needed.